src/main/java/control/TimelineController.java

```diff
@@ -36,16 +36,20 @@
     // TODO: Replace Camera Type and Scripting Project when XML functionality is available
     private final CameraType defType = new CameraType("AW-HE130 HD PTZ", "It's an IP Camera", 0.5);
 
-    // Placeholder project in lieu of XML loading
     @Getter
-    private ScriptingProject scriptingProject = new ScriptingProject("BOSS Project", 1.0);
+    private ControllerManager manager;
```

Comment:
The variable name as `controllerManager` would better represent that class that is in the value.